x/auction/types/params.go

```diff
@@ -3,96 +3,81 @@ package types
 import (
 	"bytes"
 	"fmt"
+	"time"
 
 	"github.com/cosmos/cosmos-sdk/x/params/subspace"
 )
 
 // Defaults for auction params
 const (
-	// DefaultMaxAuctionDuration max length of auction, roughly 2 days in blocks
-	DefaultMaxAuctionDuration EndTime = 2 * 24 * 3600 / 5
+	// DefaultMaxAuctionDuration max length of auction
+	DefaultMaxAuctionDuration time.Duration = 2 * 24 * time.Hour
 	// DefaultBidDuration how long an auction gets extended when someone bids, roughly 3 hours in blocks
-	DefaultMaxBidDuration EndTime = 3 * 3600 / 5
-	// DefaultStartingAuctionID what the id of the first auction will be
-	DefaultStartingAuctionID ID = ID(0)
+	DefaultBidDuration time.Duration = 3 * time.Hour
```

Comment:
I think this is too long. The main 'worry' with short bid durations is block stuffing to prevent other bids. The tradeoff with long timeouts is greater amount of locked capital required. I would start with something like 1 hour.